src/app/pages/home-page/home-page.component.scss

```diff
@@ -1,90 +1,24 @@
 @import '../../shared/constants/colors.scss';
 @import '../../shared/constants/otherStyles.scss';
 
-.wrapper-home-page {
-  height: $height-home-page;
-}
-
-.wrapper-content {
-  height: auto;
-  background: $background-content;
-}
-
-.wrapper-drop-box,
-.wrapper-description {
+.drop-box_container,
+.description_container,
+.sandbox-interviews_container {
   width: 100%;
-  padding: 6px 0 0 0;
-  display: flex;
-  justify-content: center;
-}
-
-.wrapper-drop-box {
-  height: 5%;
-}
-
-.wrapper-description {
-  height: 15%;
+  background: #fafafa;
```

Comment:
upls use constants colors

Reply:
Yeah, this color should be in our file for colors.

Reply:
done